indico/modules/events/abstracts/blueprint.py

```diff
@@ -131,6 +134,13 @@
 _bp.add_url_rule('/manage/abstracts/<int:abstract_id>/', 'manage_abstract', RHManageAbstract, methods=('GET', 'POST'))
 _bp.add_url_rule('/manage/abstracts/<int:abstract_id>/abstract.pdf', 'manage_abstract_pdf_export', RHAbstractExportPDF)
 
+# Abstract-specific
+_bp.add_url_rule('/manage/abstracts/<int:abstract_id>/comment', 'comment_abstract', RHLeaveComment, methods=('POST',))
+_bp.add_url_rule('/manage/abstracts/<int:abstract_id>/comments/<int:comment_id>/edit',
+                 'edit_abstract_comment', RHEditAbstractComment, methods=('GET', 'POST'))
+_bp.add_url_rule('/manage/abstracts/<int:abstract_id>/comments/<int:comment_id>/delete',
+                 'delete_abstract_comment', RHDeleteAbstractComment, methods=('POST',))
```

Comment:
I usually go for whatever_rest / RHWhateverREST since it's pretty much a RESTful endpoint. But in this case I wouldn't rename it. It's still only for deletion (you don't need to merge the two RHs!)